test/Interpreter/error_self_conformance.swift

```diff
@@ -0,0 +1,44 @@
+// RUN: %target-jit-run %s 2>&1 | %FileCheck %s
```

Comment:
Any reason this is a JIT test and not a build a binary and run it test?

Reply:
No, I'll fix that.